docs/create-distro.md

```diff
@@ -0,0 +1,92 @@
+          <checksumPolicy>warn</checksumPolicy>
+        </snapshots>
+      </repository>
+    </repositories>
+  </profile>
+</profiles>
+
+<activeProfiles>
+  <activeProfile>ozone</activeProfile>
+</activeProfiles>
+```
+
+### 2) Generate the archetype
+
+Use Maven's archetype tools to generate a new Ozone implementation project structure:
+
+```bash
+mvn org.apache.maven.plugins:maven-archetype-plugin:3.2.1:generate -DarchetypeArtifactId=maven-archetype -DarchetypeGroupId=com.ozonehis 
+```
+
+This will prompt you for several key variables for your Maven project:
+
+| Prompt variable    | Sample value     | Explanation                                                                                                                                                                                             |
+|--------------------|------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `distributionName` | `Ozone Gruzinia` | A name for your distribution. For example, a reference Ozone implementation for the imaginary country of Gruzinia could be named "Ozone Gruzinia".                                                      |
+| `groupId`          | `gz.moh`         | The [Maven group ID](https://maven.apache.org/guides/mini/guide-naming-conventions.html) that will be used for the implementation artifact. For "Ozone Gruzinia" this might be something like `gz.moh`. |
+| `artifactId`       | `ozone-gruzinia` | The [Maven artifact ID](https://maven.apache.org/guides/mini/guide-naming-conventions.html) that will be used for the implementation artifact. For "Ozone Gruzinia" this might be `ozone-gruzinia`.     |
+| `package`          | `gz.moh`         | A required property, but not used. To make it easy just accept the default value, which should default to be the same as the `groupId`.                                                                 |
+| `version`          | `1.0.0-SNAPSHOT` | The version number for the distribution, it starts by default at `1.0.0-SNAPSHOT`.                                                                                                                      |
+
+
+This will create a bare-bones Ozone implementation project that should look like this:
+```bash
+ozone-gruzinia/
+├── README.md
+├── config
+│     └── openmrs
+│         ├── frontend_config
+│         └── initializer_config
+├── pom.xml
+├── readme
+│     └── impl-guide.md
+└── scripts
+    ├── mvnw
+    ├── mvnw.cmd
+    ├── mvnwDebug
+    └── mvnwDebug.cmd
+```
+
+## Available commands
+
+|Action|Command|
+|:----|:----|
+|Build the distribution|<pre>./scripts/mvnw clean package</pre>|
+|Run the distribution|<pre>source target/go-to-scripts-dir.sh<br/><br/>./start-demo.sh</pre>|
+|Stop the distribution|<pre>./stop-demo.sh</pre>|
+|Destroy the distribution|<pre>./destroy-demo.sh</pre>|
```

Comment:
I'll be merging this for now, but I would like to hear from you (@ibacher, @rbuisson) about this table presentation here.

Reply:
Seems right. The last two commands do assume you're in the directory the `./start-demo.sh` script is launched from, but I don't know how to make that clear.